validator/src/main.rs

```diff
@@ -50,6 +51,10 @@ struct Args {
     /// Optional: Leviticus Validator URL
     #[arg(long, default_value = None)]
     leviticus_url: Option<String>,
+
+    /// Optional: Validator penalty
+    #[arg(long, default_value = "1000")]
+    validator_penalty: u64,
```

Comment:
this makes it so that we can only set whole-integer validator penalties, otherwise the type conversion will fail, probably good to document somewhere